developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -136,6 +136,54 @@ bundle exec rake -T
 == Installing ChromeDriver
 https://chromedriver.chromium.org/[ChromeDriver] is a separate executable that Selenium WebDriver uses to control Chrome. We use ChromeDriver to run the integration tests for Foreman.
 
+[[plugins]]
+== Installing plugins
+In order to use a plugin, you'll need to install its gem.
+
+From source code:
+[source, ruby]
+....
+cd foreman
+echo "gem '<PLUGIN_NAME>', path: '../PLUGIN_PATH'" >> bundler.d/<PLUGIN_NAME>.local.rb
+....
+
+From github:
+[source, ruby]
+....
+cd foreman
+echo "gem '<PLUGIN_NAME>', git: 'https://github.com/theforeman/<PLUGIN_NAME>.git'" >> bundler.d/<PLUGIN_NAME>.local.rb
+....
+
+Then simply run `bundle install` from foreman to install the plugin and its dependencies.
```

Comment:
```suggestion
Then run `bundle install` from foreman to install the plugin and its dependencies.
```
I'm against using the word "simply" in a guide, because its not simple, thats why there is a guide :)

Reply:
lol, no problem.. updated :)